applications/debug/unit_tests/rpc/rpc_test.c

```diff
@@ -18,6 +18,8 @@
 #include <cli/cli.h>
 #include <loader/loader.h>
 #include <protobuf_version.h>
+
+#include <FreeRTOS.h>
```

Comment:
taskENTER_CRITICAL and taskEXIT_CRITICAL can also be isolated from user code

Reply:
RPC tests in general uses FreeRTOS directly. I was hoping that @gsurkov may take a look on it later.

As for taskENTER_CRITICAL/taskEXIT_CRITICAL there is one exceptional place in furi_hal_flash, which I'd rather not disturb now. There were some issues with it.